### PR TITLE
Improvement: refactored language selector component

### PR DESCRIPTION
**Pull request by damianmuti**
I've refactored the whole language selector component, since the structure was complicated to maintain and to manipulate content inside each of the language buttons.
Now we have a cleaner solution with a list of elements that are exactly the same.

image

### CODE REVIEW

**Comment by andresmoschini** [comment]
Great! thanks @damianmuti ❤️

**Comment by andresmoschini** [comment]
:shipit:

File: assets/templates/signup.html [hunk 1]
```diff
@@ -51,9 +51,15 @@
...
-            <div class="option" id="spanish-selector">
-              <span><img src="img/spanish-flag.svg"><strong>ES</strong></span>
-              <a class="option--item" href="#"><img src="img/english-flag.svg"><strong>EN</strong></a>
+            <div class="lang-option option--selector">
+              <button class="lang--es">
+                ES
+              </button>
+            </div>
+            <div class="lang-option">
+              <button class="lang--en">
```
Comment (fcoronelmakingsense) [inline assets/templates/signup.html]: Need this in `<a>` because is a link

**Comment by GusBaamonde** [comment]
Cuando despliega la banderita si clickeas fuera no se oculta y creo que @fcoronelmakingsense necesitaba que no sean buttons, creo que prefiere links

File: assets/templates/signup.html (hunk 1 above)
Comment (andresmoschini) [reply assets/templates/signup.html]: Good catch @fcoronelmakingsense! 

@damianmuti what do you think about fixing it and also applying the related changes in webapp before merging it?

Comment (damianmuti) [reply assets/templates/signup.html]: Sure, @andresmoschini ! I've just pushed the changes to this PR 👌

Comment (andresmoschini) [reply assets/templates/signup.html]: Fo, @fcoronelmakingsense, feel free to ignore these changes in your tasks related to signup page, forgot page, etc.

**Pull request merged**